Add call for tasks page (#2)
ia-toki/apio-2020-web: call-for-tasks.md

---
layout: single-page-md
title: APIO 2020 Call for Tasks
key: call-for-tasks
---

The Asia-Pacific Informatics Olympiad (APIO) is an online IOI-like contest for the Asian and Western Pacific region.
APIO 2020 will be held on Saturday, May 9th, 2020 to Sunday, May 10th, 2020.
The host of the APIO 2020 invites everyone in the programming contests community to submit tasks for the APIO 2020 competition.
Guidelines for tasks and instructions for submission are given below.

### APIO competition tasks

> APIO tasks are basically the same as tasks in International Olympiad in Informatics, which are typically focused on the design of efficient, correct algorithms.

Reviewer: ```suggestion
APIO tasks are basically similar to the tasks in International Olympiad in Informatics, which are typically focused on the design of efficient, correct algorithms.
```

Input and output are to be kept as simple as possible.
The submission of novel tasks types not yet seen in any recent programming contests is encouraged.
The guidelines of task preparations of IOI should be considered, with no restriction.
The documents about submission of IOI tasks, which can be used as reference to create the tasks, can be found at IOI website.

To ensure a fair and interesting competition for all, tasks should satisfy the following conditions:

* The tasks should not have been seen by any potential APIO 2020 contestants.
* The tasks should not have been used in any similar competition.
* The tasks should be solvable by APIO competitors during an APIO contest round.
* The task descriptions should be unambiguous and easy to understand.
* The tasks should be culturally neutral.
* The tasks should be innovative.

### What to submit

A task submission must contain:

> Statement in English, preferably formatted in PDF with required diagrams and pictures included.

Reviewer: ```suggestion
* Statement in English, preferably formatted in PDF with required diagrams and pictures included.
```

Note that we do not wish to forbid authors of submitted tasks to be involved in other competitions and training, but we do ask them to take all necessary precautions to safeguard confidentiality.

Submitted tasks, including the description and test cases, may be subject to modifications by the organizing committee.

### Submission process

Task materials must be placed together in a single file (use .zip or .tgz for multiple files).
Email to [sc.apio2020@toki.id](mailto:sc.apio2020@toki.id) with **[Task Submission] - &lt;task title&gt;** as the subject and attach the task submission.

### Task selection process

Tasks that are not used will be kept secret, and will be returned to their submitters so that they can be used elsewhere. Note that all submitted tasks will be seen only by the host organizing committee, and the shortlisted tasks will be seen by all heads of delegations from the region. Communications among heads of delegations and hosts about the APIO tasks should be done mainly by e-mail and secured web.

> All submitted tasks will be shortlisted by the host of APIO 2020. Then, the shortlisted tasks are distributed to all heads of delegations. Hosts of delegations should provide suggestions and propose the final set of competition tasks to the host in order to make a final decision on the competition task set. If any countries need to translate the task scripts to their languages, the translated script must be submitted to the host up to three days before the competition day.

Reviewer: > Hosts of delegations should provide suggestions and propose the final set of competition tasks to the host in order to make a final decision on the competition task set.

Will this really be the case?

If yes -- what is "hosts of delegations"?

At various moments during the preparation of the APIO 2020 competition, it is possible that submitted tasks are dropped from further consideration. However, it may still be necessary to revert such decisions later. In general, all submitted tasks, whether dropped or not, should be kept secret until the end of competition.

Authors of submitted tasks will be recognized during APIO 2020 by listing their name, affiliation, and country (unless they specifically decline this).

### Summary

* Submission deadline: 15 March 2020
* Language: English
* Text format: PDF preferred
* Multiple files for one task (e.g. diagrams, solution code): in a single zip or compressed tar archive
* Multiple tasks: submit separately
* Minimum contents of submission:
  * author(s) information: name, e-mail, affiliation, country, and olympiad-role;
  * task statement;
  * description of the desired solution.
* Also recommended:

>   * solution implementations in C++ or Java;

Reviewer: Do we want to support Java?
